sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -280,27 +281,31 @@ def on_request(self, request):
                 return
 
             try:
-                _LOGGER.debug("Request URL: %r", http_request.url)
-                _LOGGER.debug("Request method: %r", http_request.method)
-                _LOGGER.debug("Request headers:")
+                log_string = "Request URL: '{}'".format(http_request.url)
```

Comment:
Why are we not looking at the multi record environment variable here but we do elsewhere?

Reply:
Because it is logger.info in HttpLoggingPolicy but _LOGGER.debug here.

You know, we are changing the log format which is technically a breaking change.

For logger.info, we added multi_record to allow user to opt in the old behavior.

https://github.com/Azure/azure-sdk-for-python/pull/20300#discussion_r697737641 for more information.
But for debug log, there is not need for that.